tests/fast/MoveKeysCycle.txt

```diff
@@ -19,4 +19,5 @@ testTitle=MoveKeysCycle
     reboot=true
     testDuration=10.0
 
-storageEngineExcludeType=-1
\ No newline at end of file
+storageEngineExcludeTypes=-1,-2
+maxTLogVersion=5
```

Comment:
At https://github.com/apple/foundationdb/blob/8a69be81f28e2a282304a6c37edc42e5e5dc97b3/fdbclient/FDBTypes.h#L752-L755, 
`V5` is used for 7.0. It is unclear if 6.3 support V5 or V4. Can you clarify that in the code? 

If `maxTLogVersion ` is configurable, shouldn't we have a test that use a different value for this parameter?

Reply:
By default, I just left it to be what `MAX_SUPPORTED_VERSION` is. The reason this PR was created in the first place was to have `maxTLogVersion` configured to another value in a different PR, but I didn't want to arbitrarily change the behaviour of any existing tests. If you think it's fine to set this to 4 for this test, then I can make that change